project/hca/config.py

```diff
@@ -90,10 +90,13 @@ def _elastic_search_client(self, host, port, timeout):
                                    use_ssl=True,
                                    verify_certs=True,
                                    connection_class=RequestsHttpConnection,
-                                   http_auth=es_auth)
+                                   http_auth=es_auth,
+                                   maxsize=16)
```

Comment:
Could you explain your reasoning behind your choice of 16 in a comment?

Reply:
Actually, could we gather the keyword args that are common to both invocations in a separate dict:

```
kwargs = dict(maxsize=16, timeout=timeout)
…
client = Elasticsearch(hosts=[dict(host=host, port=port)],
                                    use_ssl=False,
                                    **kwargs)
```